lib/Arduino_Core_Buddy/cores/arduino/USBSerial.h

```diff
@@ -18,7 +18,8 @@ class USBSerial : public Stream {
         : enabled(false)
         , isWriteOnly(false)
         , lineBuffer()
-        , lineBufferUsed(0) {}
+        , lineBufferUsed(0)
+        , lineBufferHook(nullptr) {}
```

Comment:
Rather than adding default values to constructor, it's better to add the default values to declarations (like so in this case: 
`void (*lineBufferHook)(const uint8_t *buf, int len) { nullptr };` )